learning/dashboard/Group 4/Postman.md

```diff
@@ -0,0 +1,156 @@
+
+<br>
+
+* Create your account with all the required details, or sign up with Google, as shown in the image.<br><br>
+![Javatpoint](https://i.imgur.com/UU6cI7m.png)
+
+* After signing in, select the workspace tools as per your requirement, and then click on, continue to get the startup screen.<br><br>
+![Javatpoint](https://static.javatpoint.com/tutorial/postman/images/postman-installation-and-updates5.png)
+
+* You will see the following page, and then you are ready to use Postman.<br><br>
+![Javatpoint](https://static.javatpoint.com/tutorial/postman/images/postman-installation-and-updates6.png)
+<br>
+<br>
+
+### Steps to update the Postman desktop client
+
+<br>
+
+* You will receive a notification in your Postman app if any update is available.
+
+* To download or install the latest update, click on the *Settings* icon, as shown in the image below, then go to *Update*.<br><br>
+![Javatpoint](https://static.javatpoint.com/tutorial/postman/images/postman-installation-and-updates7.png)
+
+* You can change the update settings by enabling the *Automatically download major updates* setting, as shown in the image below<br><br>
+![Javatpoint](https://static.javatpoint.com/tutorial/postman/images/postman-installation-and-updates8.png)
+
+* You can also check for the latest updates by clicking on *Check for updates*, which will show you if any new update is available.
+
+* You will have to select the update option to download and install the latest update. When the download is complete, you will see a notification asking you to restart the app to install the updated features.
+
+* It automatically downloads minor updates and bug fixes.
+
+<br>
+
+*Note that if you install Postman in a Linux environment you may not have the* Check for updates *option. This is because the updates are handled by the snap store, which automatically updates Postman on a regular basis.*
+
+<br>
+
+## Usage
+
+The following steps will show you how to use Postman after properly installing and setting it up.
+
+1. Open your Postman desktop application<br><br>
+![](https://i.imgur.com/LRI7l6I.png) 
+
+2. Open Postman and select *Workspaces*<br><br>
+![](https://i.imgur.com/JFzKUuq.png)
+
+3. In the workspace section section, click on *New*<br><br>
+![](https://i.imgur.com/mRfSIc4.png)
+
+4. After that, you will be greeted by a dialog box as seen below, click on *HTTP Request*<br><br>
+![](https://i.imgur.com/1wAT25e.png)
+
+5. You will see an input box in the *GET* section as highlighted below, you will have to put your API key or JSON in this box.<br><br>
+![](https://i.imgur.com/sRVBSoL.png)<br><br>
+Then all you need to do is press the *Send* button
+
+6. You will then be greeted with the following json response message, typically look for *200 OK* for a successful response<br><br>
+![](https://i.imgur.com/ZapU4PC.png)
+
+<br>
+Provided you followed all the above steps properly, you have now successfully used Postman to get a postive response from an API.
```

Comment:
Consider adding some basic API end point as examples and sample responses.
Add one sample API and example response for each:
- GET
- POST
- UPDATE
- DELETE

You can use any publicly available API server for the same.